src/query/service/src/interpreters/interpreter_replace.rs

```diff
@@ -212,19 +213,27 @@ impl ReplaceInterpreter {
             table_level_range_index,
             need_insert: true,
         }));
+        let target = if is_distributed {
```

Comment:
different blocks of the same segment will be executed on different nodes? will they generate different segments after replaced?